setup/vdisk.py

```diff
@@ -34,8 +35,8 @@ class VDiskSetup(object):
     def __init__(self):
         pass
 
-    @staticmethod
-    def create_snapshot(snapshot_name, vdisk_name, vpool_name, api, consistent=True, sticky=True,
+    @classmethod
+    def create_snapshot(snapshot_name, vdisk_name, vpool_name, consistent=True, sticky=True,
```

Comment:
Missing cls declaration. Snapshot_name is currently treated as the VDiskSetup pointer